packages/create-gatsby/package.json

```diff
@@ -17,15 +17,13 @@
   ],
   "devDependencies": {
     "@ascorbic/worker-threads-shim": "^1.0.0",
-    "@babel/runtime": "^7.15.4",
```

Comment:
This is also listed as a dependency, can we remove that too or is it needed?

Reply:
It was added in https://github.com/gatsbyjs/gatsby/pull/32954, I just want to leave it for now and we can revisit it later. Didn't want to break anything